sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2102,6 +2102,16 @@ class Dataset[T] private[sql](
     Limit(Literal(n), logicalPlan)
   }
 
+  /**
+   * Returns a new Dataset by skipping the first `m` rows.
```

Comment:
a typo? may be 'n'?

Reply:
Yeah. this is a minor issue. I will fix it by the way.